tests/quantizer_impl_test.py

```diff
@@ -50,7 +50,6 @@ def test_QuantizedTanh():
   qtools_quantizer = quantizer_impl.QuantizedTanh()
   qtools_quantizer.convert_qkeras_quantizer(qkeras_quantizer)
   new_quantizer = qtools_quantizer.convert_to_qkeras_quantizer(
-      symmetric=qkeras_quantizer.symmetric,
```

Comment:
Why was this line removed?

Reply:
I had removed `symmetric` along with `integer`. I have reintroduced `symmetric` in the latest commit.